<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Doctor's Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDoctorName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Reading Experiment and Report Language</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbLangReport">
     <item>
      <property name="text">
       <string>Spanish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbDemoMode">
     <property name="text">
      <string>Activate Demo Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbEnableDualMonitor">
     <property name="text">
      <string>Enable Dual Monitor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Binding</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbBindingDefault">
        <property name="text">
         <string>2 Targets - Large</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Select Number of Targets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboxBindingNumberOfTargets">
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbBindingUseNumbers">
        <property name="text">
         <string>Use numbers instead of an X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbOk</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
